cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceStateMachineTest.java

```diff
@@ -532,4 +508,8 @@ public void testStoppedRefreshToPendingProposalNotEnoughData(Vertx vertx, VertxT
                 .onComplete(result -> checkOptimizationResults(result, context, true));
 
     }
+
+    private static class State extends AbstractOperatorState {
```

Comment:
I assume this is the placeholder for if you make the underlying matcher more abstract?

Reply:
Right, if I need to override the methods in the `AbstractResourceStateMatchers`.